src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3935,6 +3941,77 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
   </Target>
 
+  <!--
+    ============================================================
+                                        _DeploymentSetClickOnceVersions
+
+    Sets ClickOnce versions
+    ============================================================
+    -->
+  <Target
+      Name="_DeploymentSetClickOnceVersions"
+      Condition="'$(GenerateClickOnceManifests)'=='true'">
+
+    <PropertyGroup>
+      <_DeploymentManifestTargetFrameworkMoniker>$(TargetFrameworkMoniker)</_DeploymentManifestTargetFrameworkMoniker>
+      <_DeploymentManifestTargetFrameworkVersion>$(TargetFrameworkVersion)</_DeploymentManifestTargetFrameworkVersion>
+    </PropertyGroup>
+
+  </Target>
+
+  <!--
+    ============================================================
+                                        _DeploymentGenerateLauncher
+
+    Generates Launcher if needed
+    ============================================================
+    -->
+  <Target
+      Name="_DeploymentGenerateLauncher"
```

Comment:
Meta question: what is the test strategy for this new scenario in general? Are you planning to add end to end tests in the SDK repo?

Reply:
We have not added any tests yet, but plan to add them for old and new scenarios. @sujitnayak are there any tests for existing (old) scenarios?

Reply:
No we don't have any existing ClickOnce tests. I am looking at adding some tests in the dotnet/sdk repo with the other PR.